cypresse2e/cypress/integration/e2eTests/header/header.spec.js

```diff
@@ -1,15 +1,29 @@
+import Header from '../../../elements/pages/Header';
```

Comment:
Rather than giving full path, we can define the import statement with only the 'page/class_name'  like

import Header from './Header';

Reply:
> Rather than giving full path, we can define the import statement with only the 'page/class_name' like
> 
> import Header from './Header';

@galeza 
please ignore this this step, this works if we don't have folders hierarchy described. Sorry for this one.

Thanks